package.json

```diff
@@ -50,9 +50,9 @@
     "karma-coverage-istanbul-reporter": "~2.1.0",
     "karma-jasmine": "~2.0.1",
     "karma-jasmine-html-reporter": "1.5.4",
-    "ng-packagr": "^15.0.3",
-    "protractor": "^7.0.0",
-    "ts-node": "~8.3.0",
-    "typescript": "~4.8.4"
+    "ng-packagr": "16.2.3",
+    "protractor": "~7.0.0",
```

Comment:
Can we get rid of protractor? If that's the case, I would take advantadge of this PR to do so.